src/drivers/code_hosting/gitlab.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local src="merge_request%5Bsource_branch%5D=${2}"
+  local dest="merge_request%5Btarget_branch%5D=${3:-master}"
```

Comment:
This is a bash feature. If the third positional argument is not defined then it is set to the string 'master'. So this sets the default target branch to 'master'.